spec/system/jobseekers/jobseekers_can_update_their_profile_from_job_applications_spec.rb

```diff
@@ -25,51 +25,17 @@
     after { logout }
 
     scenario "jobseekers can update their profile qualifications using the job application information on submission" do
```

Comment:
[Minor] think it should be "jobseekers can update their profile qualifications, employment history and training and cpds using the job application information on submission"